### ISSUE TITLE
Export  Comments

### ISSUE THREAD

**Issue by dougviana** [holistics/dbml#40]
There are some way to export the comments together my diagram?

**Comment by phuongduyphan** [comment]
Hi @dougviana ,

Currently we do support export column comments. The DBML syntax is like this: 

```
Table orders {
    status varchar [
    note: '
    💸 1 = processing, 
    ✔️ 2 = shipped, 
    ❌ 3 = cancelled,
    😔 4 = refunded
    ']
} 
```

Exported MySQL:
```
CREATE TABLE `orders` (
  `status` varchar(255) COMMENT '
    💸 1 = processing, 
    ✔️ 2 = shipped, 
    ❌ 3 = cancelled,
    😔 4 = refunded
    '
);

```

Index and table comment will be supported soon !

**Comment by phuongduyphan** [comment]
Hi @dougviana ,

We will support export comments soon ! Will keep you updated when it's out

**Comment by phuongduyphan** [comment]
Hi there,

We do support export comment for columns. The syntax is [here](https://www.dbml.org/docs/#metadata-column-notes). 

Table and index note will be supported soon !

**Comment by 4unkur** [comment]
@phuongduyphan by saying Table *notes* you mean *comments* ?

For example, I tried to convert MySQL dump to DBML:  
```sql
CREATE TABLE `assignment_scope` (
  `assignment_id` bigint unsigned NOT NULL,
  `scope_id` bigint unsigned NOT NULL
) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_unicode_ci COMMENT='My comment here';
```
and in the result, there is no `Note:` in the DBML file

**Comment by nguyenalter** [comment]
Fixed in #252
